css/style.css

```diff
@@ -1 +1,12 @@
+Body
+{
+	Background-color:white;
```

Comment:
lower case this B. Also, the page in naturally white. So adding background: white doesn't help you out. You can delete this line.